src/services/AirbenderDetails.jsx

```diff
@@ -0,0 +1,5 @@
+//handles a call to the air bender details page of a specific character
+export const getDetail = () => {
```

Comment:
you'll want to pass the CharacterId through to getDetail, so include it inside of the parens. You'll also want to add a slash after characters and before ${CharacterId}.